Edit readme

inowtak opened the pull request:
#What
Readme.mdにデータベースの下書きをした
#Why
適切なデータベースを作るため

Commit:
Edit readme

MrUrakou commented on README.md:
```diff
@@ -22,3 +22,48 @@ Things you may want to cover:
+
+|Column|Type|Options|
+|------|----|-------|
+|name|string|null: false, unique: true|
```
unique: trueにすると同姓同名の人が登録出来なくなりますね。
あと、後半に検索機能をつけるので、indexオプションをつけましょう。

MrUrakou reviewed (commented):
idはreference型にしましょう。
参考url
https://qiita.com/ryouzi/items/2682e7e8a86fd2b1ae47

Commit:
edit readme

Change the types from ‘integer’ to ‘reference’

Delete ‘unique: true’  and add ‘index :true’ in options of User table

MrUrakou commented on README.md:
```diff
@@ -22,3 +22,48 @@ Things you may want to cover:
+
+|Column|Type|Options|
+|------|----|-------|
+|user_id|reference|null: false|
```
referencesです。
外部キー制約も付けましょう。
参考url
https://qiita.com/ryouzi/items/2682e7e8a86fd2b1ae47
以下同様です。

Commit:
add 'foreign_key'
